### PR TITLE
[update-readme]

### PR DESCRIPTION
**Pull request by aeng7**
-Updated readme with simpler examples by removing Discovery, Freewheel, and OSMF plugins

### CODE REVIEW

**Comment by pilievOoyala** [comment]
LGTM

File: sample.html [hunk 1]
```diff
@@ -19,8 +17,7 @@
     "playerBrandingId": "26e2e3c1049c4e70ae08a242638b5c40",
     'debug':true,
     "skin": {
-      "config": "",
```
Comment (pilievOoyala) [inline sample.html]: Why are we removing this?

Comment (pilievOoyala) [reply sample.html]: Do we not need the iframe example?

**Comment by pilievOoyala** [comment]
@aeng7  You can merge, I just have 1 question.

File: sample.html (hunk 1 above)
Comment (aeng7) [reply sample.html]: @aldigun said to remove it. I'm assuming it's because it is outside the scope of a simple example.

**Pull request merged**